src/VisualStudio/Core/Def/Implementation/DebuggerIntelliSense/DebuggerTextView.cs

```diff
@@ -364,6 +364,25 @@ public void DisconnectFromIntellisenseControllers()
             }
         }
 
+        public void QueuePostLayoutAction(Action action) => _innerTextView.QueuePostLayoutAction(action);
+
+        public bool TryGetTextViewLines(out ITextViewLineCollection textViewLines) => _innerTextView.TryGetTextViewLines(out textViewLines);
+
+        public bool TryGetTextViewLineContainingBufferPosition(SnapshotPoint bufferPosition, out ITextViewLine textViewLine)
+        {
+            // This implementation confirms with the Editor implementation.
```

Comment:
Why doesn't this just do _innerTextView.TryGetTextViewLineContainingBufferPosition() like everything else?

Reply:
_innerTextView isn't necessarily an ITextView2

Reply:
because _innerTextView.TryGetTextViewLineContainingBufferPosition() is currently provides an incompatible signature. Its out parameter is ITextViewLineCollection or something like this.

The editor team will fix this later. But now we have a breaking change in Dev16 preview 1 that we should fix ASAP.

---
In reply to: [223910753](https://github.com/dotnet/roslyn/pull/30419#discussion_r223910753) [](ancestors = 223910753)